scholly/base/src/layouts/layout2/components/landingPage/LandingPage.js

```diff
@@ -20,13 +26,51 @@ const landingPage = () => {
           {' '}
         </title>
       </Helmet>
-      <Container fluid className="pages-top-padding">
-        <div className="landing-center-title">About Us</div>
-        <Image className="team-photo" src="https://images.pexels.com/photos/853168/pexels-photo-853168.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500" />
-        {homepageData.aboutUs.map((paragraph) => (
-          <div className="landing-center-content">{paragraph}</div>
-        ))}
-        <br />
+      <Container fluid className=" mt-3 mb-3">
+        <h2>
+          {' '}
+          Welcome to the
```

Comment:
Nit: Can we remove `the`, to me 'Welcome to Monash Association of Coding' sounds a bit better than 'Welcome to the Monash Association of Coding'.

Reply:
Or Alternatively how about titling this as `About {team name}` instead? I don't have a strong preference for either but just throwing it out there.